src/main/cms/cms_menu_gps_rescue.c

```diff
@@ -53,6 +53,11 @@ static uint16_t gpsRescueConfig_velP, gpsRescueConfig_velI, gpsRescueConfig_velD
 static uint16_t gpsRescueConfig_yawP;
 static uint16_t gpsRescueConfig_targetLandingAltitudeM;
 static uint16_t gpsRescueConfig_targetLandingDistanceM;
+static uint8_t gpsRescueConfig_altitudeMode;
+
+static const char * const osdTableAltitudeMode[] = {
+    "MAX_ALT", "FIXED_ALT", "CURRENT_ALT"
+};
```

Comment:
Instead of redefining the lookup values, reuse the same thing already defined in settings.c (`lookupTableRescueAltitudeMode[]`). Saves a little flash space.

Reply:
Ok